### ISSUE TITLE
Feature request: get location of personal dictionary file

### ISSUE THREAD

**Issue by ashton314** [rrthomas/enchant#329]
For context, I'm using [Jinx](https://github.com/minad/jinx) which is an Emacs frontend to Enchant.

I would like to know where words get saved when I add them to my personal dictionary. I'm using aspell with Nix, and it's not entirely apparent to me where those words go—they don't seem to be appearing in `~/.aspell.en.pwl`.

It would be nice if there were a little function that we could call to ask Enchant where it is storing words.

**Comment by rrthomas** [comment]
The location of personal wordlists is documented in `enchant(1)`. This is a fairly straightforward aspect of Enchant (in particular in Enchant 2.5, which no longer supports spelling providers' personal wordlists), so I don't think it's worth adding API functionality for. Effectively, it's a constant that Jinx could easily compute; in any case, "a little function that we could call" sounds to me like you mean "a little Jinx function", so I'd suggest a feature request for Jinx (which I must get around to trying some time!).

**Comment by ashton314** [comment]
Oh wonderful! Thank you! I opened [an issue for Jinx](https://github.com/minad/jinx/issues/67) a while back but I don't think Enchant 2.5 was out. I found the file with no problem. Thank you!

I noticed there were some file-local vars set when I was poking around the source code—nice to meet another Emacs user in the wild. :) Jinx is excellent and I highly recommend using it! (Works best with [Vertico](https://github.com/minad/vertico) if you're not using that already too.)
